test/WebSites/RazorWebSite/Views/PartialViewEngine/PartialMissingSection.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
```

Comment:
Can also remove almost all content from the `.cshtml` files because they aren't included in an `HttpResponseMessage`. That is, unless you go with the "`try` / `catch` in the layout file" approach I suggested [above](https://github.com/aspnet/Mvc/pull/4025/files#r51515654).